contracts/ERC-721/IERC721PoolFactory.sol

```diff
@@ -41,4 +55,31 @@ interface IERC721PoolFactory {
     ///
     /// @param asset The underlying ERC-721 asset contract address.
     function createPool(address asset) external;
+
+    /// @notice Rescue the last NFT of a pool.
+    ///
+    /// @dev Emits a {RescueLastNFT} event.
+    ///
+    /// @dev Requirements:
+    /// - Can only rescue the last NFT of a pool.
+    /// - Can only be called by the owner.
+    /// - The pool must exist.
+    ///
+    /// @param asset The underlying ERC-721 asset contract address.
+    function rescueLastNFT(address asset, address to) external;
```

Comment:
@surbhiaudichya the `@param` comment for the `to` address is missing.

Reply:
@scorpion9979 Thank you for pointing out the missing @param comment for the to address in the code. I will add it.